website/content/en/applications.adoc

```diff
@@ -3,41 +3,57 @@ title: "About Applications for FreeBSD"
-* *Software development.* A suite of development tools comes with FreeBSD, including the GNU C/C++ compiler and debugger. The LLVM-based clang suite is also provided and will eventually replace the GNU suite. Java(R) and Tcl/Tk development are also possible for example, and more esoteric programming languages like Icon work just fine, too. And FreeBSD's shared libraries have always been easy to make and use. You can also choose from a wide range of popular and powerful editors, such as Emacs and Vim.
-* *Net surfing.* A real UNIX workstation makes a great Internet surfboard. FreeBSD versions of http://www.chromium.org/Home[Chromium] and http://www.mozilla.org/firefox/[Firefox] are available for serious web users. Surf the web, publish your own web pages, read Usenet news, and send and receive email with a FreeBSD system on your desktop.
-* *Education and research.* FreeBSD makes an excellent research platform because it includes complete source code. Students and researchers of operating systems or other computer science fields can benefit greatly from such an open and well-documented system.
-* *And much more.* Accounting, action games, MIS databases, scientific visualization, video conferencing, Internet relay chat (IRC), home automation, multiuser dungeons, bulletin board systems, image scanning, and more are all real uses for FreeBSD today.
+*Browsing and email*.
+Ported web browsers include https://www.mozilla.org/firefox/[Firefox] and https://www.chromium.org/Home[Chromium].
+Send and receive email with applications such as https://www.thunderbird.net/[Thunderbird].
 
-== FreeBSD is an operating system that will grow with your needs.
+*Education and research*.
+A history of almost thirty years, widespread understanding, openness and thorough documentation are amongst the things that make FreeBSD ideal for students and researchers of operating systems and other computer sciences.
+FreeBSD makes an an excellent research platform.
 
-Though FreeBSD is free software, it is also _user supported_ software. Any questions you have can be posted to hundreds of FreeBSD developers and users simply by e-mailing the freebsd-questions@FreeBSD.org mailing list.
+*Software development*.
+FreeBSD's shared libraries are easy to make and use.
+LLVM-based clang is the default compiler.
+Also integral to the OS: GNU C/C++.
+Java(R), Tcl/Tk and more are supported.
+Editors include Emacs and Vim.
 
-FreeBSD also has a worldwide group of programmers and writers who fix bugs, add new features and document the system. Support for new devices or special features is an almost constant development process, and the team keeps a special eye out for problems which affect system stability. FreeBSD users are quite proud of not only how fast but how reliable their systems are.
+*More*!
+Accounting, action games, home automation, image scanning, Internet relay chat (IRC), MIS databases, scientific visualization, video conferencing, web page publication, and much more.
```

Comment:
Matrix and Rocket Chat should probably be mentioned, since those are used more, now, than IRC (context varies)

Reply:
Plus Discord, and so on. I'm inclined to have these relatively modern things up higher, around the _Desktop_ paragraph. 

IRC, which has been around much longer, is intentionally down low under _More!_ because (I suspect) people who know about IRC are also likely to know that FreeBSD can do IRC without being told so by a page such as this.